src/scss/trumps/_utilities.scss

```diff
@@ -100,6 +100,13 @@
     top: spacing(x2);
 }
 
+.u-absolutelyCentered {
```

Comment:
Is it not worth having this as a placeholder too? So you can use it as an extend as well as inline?

Reply:
It makes sense, but then we need to add it for all utility classes, don't we? I can do it in separate PR

Reply:
I mean't just for this one as I can see it being handy in other places where others might prefer to use extends instead of adding additional classes to the dom. It's not a biggy tho so happy as it is.